dockers/rebalancer/storage/controller/Dockerfile

```diff
@@ -0,0 +1,61 @@
+ARG GO_VERSION=latest
+ARG BASE_TAG=latest
+ARG DISTROLESS_IMAGE=gcr.io/distroless/static
+ARG DISTROLESS_IMAGE_TAG=nonroot
+ARG UPX_OPTIONS=-9
+
+FROM golang:${GO_VERSION} AS golang
+
+FROM vdaas/vald-base:${BASE_TAG} AS builder
+ARG UPX_OPTIONS
+
+ENV ORG vdaas
+ENV REPO vald
+ENV PKG rebalancer/storage/controller
+ENV APP_NAME rebalancer-storage-controller
+
+COPY --from=golang /usr/local/go $GOROOT
+
```

Comment:
Please update the builder image as the other images on master branch. @hlts2 must know that.

Reply:
fixed